bin/server.js

```diff
@@ -19,9 +19,11 @@ var Mailer = require('../mailer')(legacyMailerLog)
 
 var dbConnect = require('../lib/db_connect')()
 
+var DEFAULT_LANGUAGE = 'en'
+
 P.all(
   [
-    require('../translator')(config.get('locales')),
+    require('../translator')(config.get('locales'), DEFAULT_LANGUAGE),
```

Comment:
Looks like `DEFAULT_LANGUAGE` is a config value, can we use? https://github.com/mozilla/fxa-auth-mailer/blob/master/config.js#L81